### DISCUSSION TITLE
Typing a combination of mod acronyms to select multiple mods

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#30331] [Ideas] [answered]
In mod select screen, in addition to https://github.com/ppy/osu/pull/30330, it would be nice to be able to type something like "hdhr", press enter, and have both the Hidden and Hard Rock mods selected as a result.

**Comment by jhk2601** [comment]
This is true, tried to implement this quickly but it requires a more comprehensive change because searching "hdhr" hides every column due to no mod name containing hdhr. Not in scope of 30330 but should be added to the todo list along with handling for presets. Is making an issue the right thing to do to note that down?

**Reply by Tenexxt** [reply]
Maybe a way to make the game see that it's supposed to search through acronyms and not whole names is to add a symbol at the start of the search, so for an example when you want to search for acronyms of HDHR, you type `HD HR (the symbol can be changed, this is an example of how it could [maybe] work)

**Reply by jhk2601** [reply]
no this is flawed, its barely even easier at that point than just pressing "dt, enter, escape, hr" etc. and its 100x less intuitive. Mod presets exist if you want to have a specific combo used all the time.

**Comment by peppy** [comment, marked as answer]
Just type them one by one.

HD enter escape HR enter escape

**Reply by vatei** [reply]
Sure of course, just thought it was a nice QoL thing
